### ISSUE TITLE
Verify() returning invalid progress?

### ISSUE THREAD

**Issue by kamil-cupial** [ied206/ManagedWimLib#2]
Hi,

The following code:
```
if (x == ProgressMsg.VerifyStreams)
{
    VerifyStreamsProgress p = (VerifyStreamsProgress)info;
    Logger.LogRaw($"[{p.WimFile}] {p.CurrentBytes}/{p.TotalBytes} ({p.CurrentStreams}/{p.TotalStreams})\n", false);
}
```
```
using (ManagedWimLib.Wim wim = ManagedWimLib.Wim.OpenWim(fileNameWithPath, ManagedWimLib.OpenFlags.CheckIntegrity, callback))
{
                wim.VerifyWim();
} 
```

Prints the same values each time the callback is called and they are not updated.
Verification itself works though, as both a valid and incomplete .wims were tested.

![obraz](https://user-images.githubusercontent.com/68418864/94419891-a6759780-0183-11eb-868c-3a0289edab03.png)

Is there something wrong with my usage?

**Comment by javmarquez13** [comment]
> Hi,
> 
> The following code:
> 
> ```
> if (x == ProgressMsg.VerifyStreams)
> {
>     VerifyStreamsProgress p = (VerifyStreamsProgress)info;
>     Logger.LogRaw($"[{p.WimFile}] {p.CurrentBytes}/{p.TotalBytes} ({p.CurrentStreams}/{p.TotalStreams})\n", false);
> }
> ```
> 
> ```
> using (ManagedWimLib.Wim wim = ManagedWimLib.Wim.OpenWim(fileNameWithPath, ManagedWimLib.OpenFlags.CheckIntegrity, callback))
> {
>                 wim.VerifyWim();
> } 
> ```
> 
> Prints the same values each time the callback is called and they are not updated.
> Verification itself works though, as both a valid and incomplete .wims were tested.
> 
> ![obraz](https://user-images.githubusercontent.com/68418864/94419891-a6759780-0183-11eb-868c-3a0289edab03.png)
> 
> Is there something wrong with my usage?

How to resolve issue Kamil


![image](https://user-images.githubusercontent.com/28401144/114241352-3e7be700-9946-11eb-9e1f-15e473a65bc8.png)



Thanks in advance

**Comment by ied206** [comment]
@kamil-cupial Thanks, there was a bug on `VerifyStreamsProgress`. `uint64_t` must have been translated into `ulong` instead of `uint`.

```c
/** Valid on messages ::WIMLIB_PROGRESS_MSG_VERIFY_STREAMS.  */
struct wimlib_progress_info_verify_streams {
	const wimlib_tchar *wimfile;
	uint64_t total_streams;
	uint64_t total_bytes;
	uint64_t completed_streams;
	uint64_t completed_bytes;
} verify_streams;
```

```csharp
[StructLayout(LayoutKind.Sequential)]
public class VerifyStreamsProgress
{
	public string WimFile => Wim.Lib.PtrToStringAuto(_wimFilePtr);
	private IntPtr _wimFilePtr;
	public uint TotalStreams; // Should be ulong
	public uint TotalBytes; // Should be ulong
	public uint CurrentStreams; // Should be ulong
	public uint CurrentBytes; // Should be ulong
}
```

**Comment by ied206** [comment]
Fix applied on [v2.2.0](https://github.com/ied206/ManagedWimLib/releases/tag/v2.2.0) release.
